src/common/paths/buckets.yml

```diff
@@ -3,49 +3,120 @@ get:
   tags:
     - Buckets
   summary: List all buckets
+  description: |
+    Retrieves a list of [buckets]({{% INFLUXDB_DOCS_URL %}}/cloud/reference/glossary/#bucket).
+
+    To limit which buckets are returned, pass query parameters in your request.
+    If no query parameters are passed, InfluxDB returns all buckets up to the
+    default `limit`.
+
+    #### Limitations
+
+    - Paging with an `offset` greater than the number of records will result in
+    an empty list of buckets--for example:
+
+      The following request is paging to the 50th record, but the user only has
+      10 buckets.
+
+      ```sh
+      $ curl --request GET "INFLUX_URL/api/v2/scripts?limit=1&offset=50"
+
+      $ {
+          "links": {
+              "prev": "/api/v2/buckets?descending=false\u0026limit=1\u0026offset=49\u0026orgID=ORG_ID",
+              "self": "/api/v2/buckets?descending=false\u0026limit=1\u0026offset=50\u0026orgID=ORG_ID"
+          },
+          "buckets": []
+        }
+      ```
+
+    #### Related Guides
+
+    - [Manage buckets]({{% INFLUXDB_DOCS_URL %}}/cloud/organizations/buckets/)
   parameters:
     - $ref: "../parameters/TraceSpan.yml"
     - $ref: "../parameters/Offset.yml"
     - $ref: "../parameters/Limit.yml"
     - $ref: "../parameters/After.yml"
     - in: query
       name: org
-      description: The name of the organization.
+      description: |
+        Organization name.
+        The name of the organization.
       schema:
         type: string
     - in: query
       name: orgID
-      description: The organization ID.
+      description: |
+        Organization ID.
+        The organization ID.
       schema:
         type: string
     - in: query
       name: name
-      description: Only returns buckets with a specific name.
+      description: |
+        Bucket name.
+        Only returns buckets with a specific name.
       schema:
         type: string
     - in: query
       name: id
-      description: Only returns buckets with a specific ID.
+      description: |
+        Bucket ID.
+        Only returns buckets with a specific ID.
       schema:
         type: string
   responses:
     "200":
-      description: A list of buckets
+      description: |
+        Success.
+        The response body contains a list of buckets and URI links to adjacent
+        buckets.
```

Comment:
```suggestion
        The response body contains a list of buckets and URIs for additional paged results.
```
?

Reply:
Changed this to `The response body contains a list of buckets.` and provided additional description regarding the links in `Links.yml`